Packs/Automox/Integrations/Automox/Automox.py

```diff
@@ -0,0 +1,612 @@
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this  implementation, no special attributes defined
+    """
+
+    def action_on_vulnerability_sync_batch(self, org_id: int, batch_id: int, action: string):
+        url_suffix = f"/orgs/{org_id}/tasks/batches/{batch_id}"
+
+        if action == "accept":
+            url_suffix += "/accept"
+        elif action == "reject":
+            url_suffix += "/reject"
+        else:
+            raise ValueError("Action argument must be a string equal to either 'accept' or 'reject'")
+
+        return self._http_request(
+            method='POST',
+            url_suffix=url_suffix,
+        )
+
+    def action_on_vulnerability_sync_task(self, org_id: int, task_id: int, action: string):
+        payload = {
+            "action": action
+        }
+
+        return self._http_request(
+            method='PATCH',
+            url_suffix=f"/orgs/{org_id}/tasks/{task_id}",
+            payload=payload,
+        )
+
+    def delete_device(self, org_id: int, device_id: int) -> List[Dict]:
+        params = {
+            "o" : org_id
+        }
+
+        return self._http_request(
+            method='DELETE',
+            url_suffix='/servers/{device_id}',
+            params=params
+        )
+
+    def delete_group(self, org_id: int, group_id: int) -> List[Dict]:
+        params = {
+            "o" : org_id
+        }
+
+        return self._http_request(
+            method='DELETE',
+            url_suffix='/servergroups/{group_id}',
+            params=params
+        )
+
+    def get_vulnerability_sync_batch(self, org_id: int, batch_id: int) -> List[Dict]:
+        return self._http_request(
+            method="GET",
+            url_suffix=f"/orgs/{org_id}/tasks/batches/{batch_id}",
+        )
+
+    def list_devices(self, org_id: int, group_id: int) -> List[Dict]:
+        params = {
+            "o": org_id,
+            "groupId": group_id
+        }
+
+        return self._http_request(
+            method='GET',
+            url_suffix='/servers',
+            params=params
+        )
+
+    def list_groups(self, org_id: int) -> List[Dict]:
+        params = {
+            "o" : org_id
+        }
+
+        return self._http_request(
+            method='GET',
+            url_suffix='/servergroups',
+            params=params
+        )
+
+    def list_organization_users(self, org_id:  int) -> List[Dict]:
+        params = {
+            "o": org_id
+        }
+
+        return self._http_request(
+            method='GET',
+            url_suffix='/users',
+            params=params
+        )
+
+    def list_organizations(self) -> List[Dict]:
+        return self._http_request(
+            method='GET',
+            url_suffix="/orgs",
+        )
+
+    def list_policies(self, org_id, limit, page) -> List[Dict]:
+        params = {
+            "limit" : limit,
```

Comment:
Another situation where constants might be nice. `PARAM_LIMIT = 'limit'`, etc...I find it's handy when you're going to be reusing the same ones a lot